src/game/Spells/SpellAuras.cpp

```diff
@@ -8430,70 +8433,69 @@ bool _IsExclusiveSpellAura(SpellEntry const* spellproto, SpellEffectIndex eff, A
-        case 20572: //Le racial Orc (ID 20572)
-        case 17038: //l'Eau des Tombe-Hiver (ID 17038)
-        case 16329: //le Juju's Might (ID 16329)
-        case 25891: //le buff du Bijou Choc de Terre (ID : 25891)
-        case 18264: // Charge du maitre (baton epique scholo)
-        case 12022: // Chapeau pirate
-        case 22817: // PA HT Nord
-        // Aura de precision (hunt)
-        case 19506:
-        case 20905:
-        case 20906:
-        case 18262: // Pierre � Aiguiser El�mentaire (+2% crit)
-        case 24932: // chef de la Meute
-        case 24907: // aura S�l�nien
-        case 22888: // Buff Onyxia
-        case 15366: // Buff Felwood
-        case 22820: // HT +3% crit sorts
+        // Blasted lands item buffs
+        case 10693: // Spiritual Domination
+        case 10691: // Spiritual Domination
+        case 10668: // Spirit of Boar
+        case 10671: // Spirit of Boar
+        case 10667: // Rage of Ages
+        case 10670: // Rage of Ages
+        case 10669: // Strike of the Scorpok
+        case 10672: // Strike of the Scorpok
+        case 10692: // Infallible Mind
+        case 10690: // Infallible Mind
+        case 24382: // Spirit of Zanza
+        // Alcohols
+        case 25804: // Rumsey Rum Black Label
+        case 20875: // Rumsey Rum
+        case 25722: // Rumsey Rum Dark
+        case 25037: // Rumsey Rum Light
+        case 22789: // Gordok Green Grog
+        case 22790: // Kreeg's Stout Beatdown
+        case 6114:  // Raptor Punch
+        case 5020:  // Stormstout
+        case 5021:  // Trog Ale
+        case 23179: // Taint of Shadow
+        case 20007: // Holy Strength
+        case 20572: // Blood Fury
+        case 17038: // Winterfall Firewater
+        case 16329: // Juju Might
+        case 25891: // Earthstrike
+        case 18264: // Headmaster's Charge
+        case 12022: // Admiral's Hat
+        case 22817: // Fengus' Ferocity
+        case 19506: // Trueshot Aura
+        case 20905: // Trueshot Aura
+        case 20906: // Trueshot Aura
+        case 18262: // Call Bloodshot
+        case 24932: // Leader of the Pack
+        case 24907: // Moonkin Aura
+        case 22888: // Rallying Cry of the Dragonslayer
+        case 15366: // Songflower Serenade
+        case 22820: // Slip'kik's Savvy
         case 17628: // Supreme Power
-        case 22730: // Bouffe +10 Intell
-        case 18141: // Bouffe +10 Esprit
-        case 18125: // Bouffe +10 Force
-        case 18192: // Bouffe +10 Agility
-        case 18191: // Bouffe +10 Endu
-        case 25661: // Bouffe +25 Endu
-        case 24427: // Diamond Flask
-        case 17528: // Mighty Rage Potion
+        case 22730: // Increased Intellect
+        case 18141: // Blessed Sunfruit Juice
+        case 18125: // Blessed Sunfruit
+        case 18192: // Increased Agility
+        case 18191: // Increased Stamina
+        case 25661: // Increased Stamina
+        case 24427: // Toasty
```

Comment:
Why was this changed to Toasty? Spell 24427 is Diamond Flask.
There is no spell named Toasty in vanilla.

Reply:
I pulled the data for these comments straight from the vmangos database. It looks like it was called Toasty in one of the older builds. I can try to write a query to only pull the latest names if that's most useful.

| name | build |
|--------|--------|
| Toasty | 4695 |
| Diamond Flask | 4878 | 
| Diamond Flask | 5302 |